tests/core/data_model/streams/test_data_stream.py

```diff
@@ -72,571 +169,530 @@ def test_data_stream_from_jsonl_is_pickleable(tmp_path):
     assert pre_pickle_vals == post_pickle_vals
 
 
-class TestDataStream(TestCaseBase):
-    def setUp(self):
-        self.list_data_stream = core_dm.DataStream.from_iterable(
-            ["hello", "world", "!"]
+def test_bad_json_stream(tmp_path):
```

Comment:
We'll probably want this for a bad multipart too

Reply:
added!